### PR TITLE
Migrate to v10

### PR DESCRIPTION
**Pull request by nastasi-oq**
Create a "destructive" migration to move to version 10 of KeyDataset and Tag list.
Add new `/stats` view to populate homepage quickly.
Tests are green here: https://ci.openquake.org/job/zdevel_open-risk-data-dashboard/62/

### CODE REVIEW

File: backend/ordd_api/helpers/load_content_script.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    us = User.objects.create_user(
+        username='normal_user', password=ORDD_ADMIN_PASSWORD,
+        first_name='Giuseppe', last_name='Verdi')
+    us.save()
+    us.profile.title = 'Maestro'
+    us.profile.institution = 'Conservatorio di Busseto'
+    us.profile.save()
+
+if __name__ == "__main__":
+    main()
```
Comment (daniviga) [inline backend/ordd_api/helpers/load_content_script.py]: If this is intended to be used also manually please provide some help (i.e. `ORDD_ADMIN_PASSWORD` must be set)

Comment (daniviga) [inline backend/ordd_api/helpers/load_content_script.py]: If this is intended to be used also manually please provide some help (i.e. `ORDD_ADMIN_PASSWORD` must be set)

File: backend/ordd_api/migrations/0017_migrate_content_to_v10.py [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        print("  CHECK MISSING TAGGROUPS")
+        for keydataset in kd:
+            taggroup = keydataset[3]
+            print("  check [%s]" % taggroup)
+            if taggroup == "":
+                continue
+            # try:
+            item = KeyTagGroup.objects.using(db_alias).get(
+                name=taggroup)
+            # except KeyTagGroup.DoesNotExist:
```
Comment (daniviga) [inline backend/ordd_api/migrations/0017_migrate_content_to_v10.py]: This kind of exception does never happens?

Comment (nastasi-oq) [reply backend/ordd_api/migrations/0017_migrate_content_to_v10.py]: If it happens is correct that it breaks, remove the commented part at all.

File: backend/ordd_api/helpers/load_content_script.py (hunk 1 above)
Comment (nastasi-oq) [reply backend/ordd_api/helpers/load_content_script.py]: Only by migration, it remains in this form just to change less as possible.

**Pull request merged**